regions/shapes/tests/test_circle.py

```diff
@@ -59,6 +63,24 @@ def test_pix_sky_roundtrip(self):
         assert reg_new.meta['text'] != self.reg.meta['text']
         assert reg_new.visual['color'] != self.reg.visual['color']
 
+    def test_sph_transformation(self, wcs):
```

Comment:
This should hopefully be a simple search-and-replace job, but for clarity I wonder if you should replace ``test_sph_transformation`` by ``test_to_spherical_sky``